lib/app.js

```diff
@@ -15,29 +15,78 @@ import {
 } from 'otp-react-redux'
 
 import customIcons from './icons/index'
+const otpConfig = require('./config.yml')
 
 export default class TrimetWebapp extends Component {
-  render () {
-
-    const itineraryFooter = (
-      <div className='disclaimer'>
-        Before you go, check <a href='https://trimet.org/#/tracker/' target='_blank'>TransitTracker</a> for real-time arrival information and any Service Alerts that may affect your trip. You can also <a href='https://trimet.org/tools/transittracker-bytext.htm' target='_blank'>text your Stop ID to 27299</a> or call 503-238-RIDE (7433), option 1. Times and directions are for planning purposes only, and can be affected by traffic, road conditions, detours and other factors.
-        <div className='link-row'>
-          <a href='https://trimet.org/legal/index.htm' target='_blank'>Terms of Use</a> • <a href='https://trimet.org/legal/privacy.htm' target='_blank'>Privacy Policy</a>
-        </div>
-      </div>
-    )
+  /**
+   * Get branding ID from config file. Defaults to TriMet.
+   */
+  _getBranding = () => otpConfig.branding || 'trimet'
 
+  /**
+   * Get itinerary footer for specified branding ID.
+   */
+  _getItinFooter = branding => {
+    switch (branding) {
+      case 'trimet':
```

Comment:
This switch statement here makes me a bit uncomfortable. In each case block, there is very application-specific logic. I don't think this model is sustainable. Are we going to add like 50 different sections in here for each deployment? This makes me think that forking this project may be a better idea.

Reply:
I really don't think a few lines of code that swap out an itinerary footer warrants forking this project. I think this footer will be rarely used and if that's not the case, we can refactor so that the content (e.g. disclaimer and an array of links) come from the config file. It probably doesn't even make sense that this bit of code is here and not in the otp-rr library.

Basically, my take is that this approach handles the vast majority of the configuration we're likely to need. I definitely don't want to move in a direction that favors jerry-rigging in order to avoid forking the repo, but I don't think we've tipped that scale by any means yet.